File: src/main/java/com/hank/guess/Game.java
package com.hank.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10)+1; //產生1-10的數字
        System.out.println("secret: " + secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while(number != secret) {
            System.out.print("Please enter a number: ");
            number = scanner.nextInt();
            if (number < secret){
                System.out.println("Higher");
            } else if (number > secret) {
                System.out.println("Lower");
            } else {
                System.out.println("Great,the number is: " + number);
            }
        }

/*        int i = 0;
        do {
            System.out.println(i);
            i++;
        } while(i < 3);*/
    }
}

File: src/main/kotlin/com/hank/Game.kt
package com.hank

import java.util.Random
import java.util.Scanner

fun main() {
    val secret = Random().nextInt(10)+1
    println("secret: $secret")
//    val scanner = Scanner(System.`in`)
    var number:Int = 0
        while(number != secret){
            print("Please enter a number:")
//            number = scanner.nextInt()
            number = readLine()!!.toInt()
            if (number > secret){
                println("Lower")
            } else if (number < secret) {
                println("Higher")
            } else {
                println("Greate,the number is $number")
            }
        }
}

